AlphaWallet/TokenScriptClient/Coordinators/AssetDefinitionStoreCoordinator.swift

```diff
@@ -128,7 +166,8 @@ class AssetDefinitionStoreCoordinator: Coordinator {
             isTokenScriptOrXml = true
         case .unsupportedTokenScriptVersion:
             try? FileManager.default.removeItem(at: url)
-            delegate?.show(error: SchemaCheckError(msg: R.string.localizable.tokenScriptNotSupportedSchemaError()), for: self)
+            delegate?.didHandleOpenURL(in: self, with: .failure(OpenURLError.unsupportedTokenScriptVersion))
+
             return true
         case .unknownXml:
             try? FileManager.default.removeItem(at: url)
```

Comment:
We'll have to see if `.unknownXML` and `.others` get triggered for XML files. We can AirDrop .css, files etc, but we really only want to show error (or success) messages for .xml or .tsml files.

Reply:
so for cases `.unknownXML` and `.others` i need to display error?

Reply:
Leave it for now. Sounds complicated. Most important is to show a confirmation when it works.